tools/ssl-create.sh

```diff
@@ -87,16 +87,15 @@ if [ ! -r "${PEM_FINAL_CERT}" ] || [ ! -s "${PEM_FINAL_CERT}" ] || \
 
     # Generate self-signed cert with a new key and other data
     openssl req -x509 -sha256 -newkey rsa:2048 \
-        -keyout "${PEM_TMP_KEY}" -out "${PEM_TMP_CERT}" \
+        -keyout "${PEM_KEY}" -out "${PEM_CERT}" \
         -days 365 -nodes -subj "/CN=${LOCAL_HOSTNAME}" \
     || exit $?
 
```

Comment:
The private key (and final cert with it inside) should be chown'ed *and* chmod'ed to not let users other than the www-data and/or some appropriate group members to access it (need-to-know basis).